modules/rosa-list-objects.adoc

```diff
@@ -0,0 +1,443 @@
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+
+.Example
+List all of the machine pools on a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa list machinepools --cluster=mycluster
+----
+
+[id="rosa-list-regions_{context}"]
+== list regions
+
+Lists all of the available regions for the current AWS account.
+
+.Syntax
+[source,terminal]
+----
+$ rosa list regions [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|--multi-az
+|Lists regions that support for multiple availability zones.
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+
+.Example
+List all of the available regions.
+
+[source,terminal]
+----
+$ rosa list regions
+----
+
+[id="rosa-list-upgrades_{context}"]
+== list upgrades
+
+Lists all available and scheduled cluster version upgrades.
+
+.Syntax
+[source,terminal]
+----
+$ rosa list upgrades [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster for which the available upgrades will be listed.
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+
+.Example
+List all of the available upgrades for a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa list upgrades --cluster=mycluster
+----
+
+[id="rosa-list-users_{context}"]
+== list users
+Lists the cluster administrator and dedicated administrator users for a specified cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa list users [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster for which the cluster administrators will be listed.
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+
+.Example
+List all of the cluster administrators and dedicated administrators for a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa list users --cluster=mycluster
+----
+
+[id="rosa-list-versions_{context}"]
+== list versions
+
+Lists all of the OpenShift versions that are available for creating a cluster.
```

Comment:
Same about hardcoding OpenShift (do a search for "OpenShift" in the PR and make sure to check for others. "OpenShift CLI" is okay).

Reply:
Cannot change due to current attribute limitations.